.github/workflows/gitleaks.yml

```
@@ -2,7 +2,10 @@ name: Check for leaks in secrets
 on: 
     pull_request:
       branches:
-        - main
+        - 'main'
```

yamllint (1.35.1)

[warning] wrong indentation: expected 10 but found 8

(indentation)